api/turing/config/example.yaml

```diff
@@ -104,7 +104,9 @@ TuringUIConfig:
   Homepage: /turing
  
 # File path to find OpenAPI v2 specfication file
```

Comment:
This should be v3 ?

Reply:
The description should definitely be changed, but it not necessarily need to be `v3`, because currently both v2 and v3 are supported. More explanations are here: https://github.com/gojek/turing/pull/60/files/bfe9b89b7e6e9ee1e8d9a06bddab56a3eec3ca90#r643222969